contracts/script/CombineAllocs.s.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Script, console} from "forge-std/Script.sol";
+
+contract CombineAllocs is Script {
+    function setUp() public {}
+
+    function run(string[] memory allocsPaths, string memory outputPath) public {
+        for (uint256 i = 0; i < allocsPaths.length; i++) {
+            vm.loadAllocs(allocsPaths[i]);
+        }
+        vm.dumpState(outputPath);
+    }
+}
```

Comment:
this is the approach suggested in https://github.com/ethereum-optimism/optimism/pull/10777#issuecomment-2159022947 for combining multiple l1 deployments